### DISCUSSION TITLE
LiDAR support

### DISCUSSION THREAD

**Discussion by charlieforward9** [olexale/arkit_flutter_plugin#234] [Q&A]
Hello!

I am curious about this plugins ability to measure the distance of an object from the device itself, using the LiDAR sensor in the Pro model iPhones. 

Is this currently supported? If so, what is it called, I scanned the examples and did not immediately notice this as a feature.

My use case includes getting the distance of somebody's foot and head from the device and using these distances to compute a anterior/posterior sway degree.

Thank you!

**Comment by olexale** [comment]
Hey,
To my knowledge, there is no explicit API or "toggle" for the LiDAR sensor — ARKit manages its use based on the selected configuration and hardware capabilities.

**Reply by charlieforward9** [reply]
@olexale thanks for the insight.

From the sdk docs on how to get LiDAR, it looks like it should be connected to the camera package anyways.

How would you go about getting LiDAR support in an application?

**Reply by olexale** [reply]
Mind sharing the link to the docs or, even better, a swift example of what you are trying to get?

**Reply by charlieforward9** [reply]
@olexale  My goal is to get the depth data as detailed in [this documentation](https://developer.apple.com/documentation/avfoundation/additional_data_capture/capturing_depth_using_the_lidar_camera)

My use case is to combine the depth data with detected points on a person in the frame

**Reply by olexale** [reply]
Ah, now I get it. That's not ARKit, hence this API is not covered by this package.

**Reply by charlieforward9** [reply]
@olexale you seem to be quite knowledgeable in the space of a flutter. Could you give me any idea or insight into starting with the LiDAR API or attaching it to the camera package?

Amy help / comments are appreciated.
